h/static/scripts/group-list-controller.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+angular.module('h').controller(
```

Comment:
This sounds good but I might need to see an example, having trouble getting it to work..